src/OperationMath.php

```diff
@@ -59,10 +59,4 @@ abstract class OperationMath
     const LOG10 = 'log10';
     const EXP = 'exp';
     const EXP10 = 'exp10';
-    const SINH = 'sinh';
-    const COSH = 'cosh';
-    const TANH = 'tanh';
-    const ASINH = 'asinh';
-    const ACOSH = 'acosh';
-    const ATANH = 'atanh';
```

Comment:
This is odd, but we can regenerate this easily later.

Reply:
Just to track this here, I've double-checked this change and got the latest pyvips installed and used the documented command to generate the bindings. I'm not sure why these were removed

Reply:
Let's regenerate the enums after merge.